### ISSUE TITLE
Make Fakemon transmutation possible with some limitations

### ISSUE THREAD

**Issue by Fenris2142** [TwitchPlaysPokemon/public#135]
Discussion starts here (devcord/#pbr):

[https://discord.com/channels/333356453928894466/333366441078423557/844267739065221150](url)

M4 doesn't like the Idea.

Chaos_lord2:
I personally like the idea of fakemons being transmutable under the condition that only 1 can be in the pool at a time, and they always make another in thier class. Maybe have a minimum material rarity if you want to have them keep more value

---

Since I don't know who has the power to decide these things, I'm creating an issue now.

**Comment by red031000** [comment]
if a fakemon transmutes into a fakemon that's fine, but I don't think there should be "crossing" (i.e. all normal badges shouldn't be able to produce a fakemon, and a fakemon should always be produced if one is put in)

**Comment by aaaaaa123456789** [comment]
Considering how rare fakemon are, and how easy it is to sell them for amounts that will buy badges that are easily transmuted, why is there a need for this?

**Comment by Fenris2142** [comment]
Not sure if I understand your question correctly so I might make the wrong arguments now. I'm still gonna try.

First of all fakemon badges, as they are now, are an isolated system. No fakemon badges get created, no fakemon badges get destroyed. This won't change if we implement fakemon transmutation like chaos or red proposed (which is the preferred option anyway). The number of fakemon badges in circulation always stays the same.

Second, the value of fakemons is highly dependant on how memorable the Pokemon is or was. There is a handful of fakemons which sell for high prices like Machu, Phancero, Violo. The majority of fakemons however sells for... well even less than an Uncommon Gen 4 badge. Somewhat recent example:
[https://tpp.chat/id/1617521776928/#1617521776928](url)

These 2 fakemons were on sale for T10 for some time until I decided to buy them. Badges with 35 and 68 existing, other "trash" fakemon badges are usually sold in the T20-T30 range (still less than an uncommon Gen 4).

Thing is, people don't know what to do with these fakemons. They win them, they don't want to collect them, they can't transmute or break them, so they put em on the market for trash prices. I'm pretty sure that if Chaos didn't buy most badges for T30 the price would have dropped even more.

So right now the fakemon market is kinda stale. There is still high demand for some badges like Machu and Violo (exploitable Noisemaker) with no way of getting more of them atm. This is the complete opposite of the regular natdex badge market, where any badge can always be transmuted. To fix this and to give people something to do I proposed to make fakemons "reformable" which was then changed to "transmuteable" by chaos. An idea I can also live with.

Lastly I want to add my personal motive why I proposed this change. I'm actually collecting fakemon badges and as of writing this, I have collected all distinct fakemons that were given out using the new fakemon distribution system. Pokemon Vega, where most of these badges were distributed was one hell of a run for me. I had no regular sleep scedule for a week. The FOMO was real and I kinda don't want to experience this again during a potential Sage or Dreams or Prism revisit or whatever run. Which is why I think that fakemon transmutation is a cool idea.

**Comment by red031000** [comment]
I think it's worth mentioning that my idea only really works when there's a limit of one fakemon badge input at a time, otherwise it can result in loss of fakemon badges

**Comment by aaaaaa123456789** [comment]
In that case, the answer is probably not transmutation (which is meant to take multiple inputs and accumulate rarities by design), but rather a similar "badge swap" feature that only works for fakemon badges and does a one-for-one swap — perhaps taking tokens to improve rarity instead of additional badges.

**Comment by Fenris2142** [comment]
that was my initial proposal (see discussion linked above). Chaos said he'd rather have a transmutation with limitations to have an additional badge sink. Thats a good idea as well. 
Making it transmutable solves a lot of stuff already, like availability of this system.

**Comment by Megamaw** [comment]
i agree with fenris. with fakemon, FOMO is a real concern, and should probably be addressed. the incentive to completely destroy my body for phancero, which i coveted for years, during chatty crystal was real, knowing it was permanently missable.

and anything where FOMO has to be addressed, generally, my take is 'don't make it a concern'.

this would also add value to other fakemon badges, and if implemented to include mons we missed, make any runs with fakemon a little less painful to fail completing the dex of.

i also agree with red's take this should be approached as  a regular badge sink, and not a fakemon badge sink.
